rcl/include/rcl/validate_security_context_name.h

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCL__VALIDATE_SECURITY_CONTEXT_NAME_H_
+#define RCL__VALIDATE_SECURITY_CONTEXT_NAME_H_
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include "rmw/validate_namespace.h"
+#include "rmw/validate_node_name.h"
+
+#include "rcl/macros.h"
+#include "rcl/types.h"
+#include "rcl/visibility_control.h"
+
+#define RCL_SECURITY_CONTEXT_NAME_VALID RMW_NAMESPACE_VALID
+#define RCL_SECURITY_CONTEXT_NAME_INVALID_IS_EMPTY_STRING RMW_NAMESPACE_INVALID_IS_EMPTY_STRING
+#define RCL_SECURITY_CONTEXT_NAME_INVALID_NOT_ABSOLUTE RMW_NAMESPACE_INVALID_NOT_ABSOLUTE
+#define RCL_SECURITY_CONTEXT_NAME_INVALID_ENDS_WITH_FORWARD_SLASH \
+  RMW_NAMESPACE_INVALID_ENDS_WITH_FORWARD_SLASH
+#define RCL_SECURITY_CONTEXT_NAME_INVALID_CONTAINS_UNALLOWED_CHARACTERS \
+  RMW_NAMESPACE_INVALID_CONTAINS_UNALLOWED_CHARACTERS
+#define RCL_SECURITY_CONTEXT_NAME_INVALID_CONTAINS_REPEATED_FORWARD_SLASH \
+  RMW_NAMESPACE_INVALID_CONTAINS_REPEATED_FORWARD_SLASH
+#define RCL_SECURITY_CONTEXT_NAME_INVALID_NAME_TOKEN_STARTS_WITH_NUMBER \
+  RMW_NAMESPACE_INVALID_NAME_TOKEN_STARTS_WITH_NUMBER
+#define RCL_SECURITY_CONTEXT_NAME_INVALID_TOO_LONG RMW_NAMESPACE_INVALID_TOO_LONG
+
+#define RCL_SECURITY_CONTEXT_NAME_MAX_LENGTH RMW_NODE_NAME_MAX_NAME_LENGTH
+
+/// Determine if a given security context name is valid.
+/**
+ * /sa The same rules as rmw_validate_namespace are used.
```

Comment:
@ivanpauno same comment as elsewhere about using `\sa` with arguments other than references. Also, that slash is backwards :sweat_smile:

Reply:
Done in https://github.com/ros2/rcl/pull/515/commits/44a15be05484088c78a4e7cde08a021356757160.